pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -192,6 +209,23 @@ def _add_prefix(
         count = seen_optimizer_types[optimizer_cls]
         return name + f'-{count - 1}' if count > 1 else name
 
+    def _add_suffix(self, name: str, param_groups: List[Dict], param_group_index: int, use_names: bool = True) -> str:
+        if len(param_groups) > 1:
+            if not use_names:
+                return f'{name}/pg{param_group_index+1}'
+            else:
+                pg_name = param_groups[param_group_index].get('name', f'pg{param_group_index+1}')
+                return f'{name}/{pg_name}'
+        elif use_names:
+            pg_name = param_groups[param_group_index].get('name')
+            return f'{name}/{pg_name}' if pg_name else name
+        else:
+            return name
+
+    def _duplicate_param_group_names(self, param_groups: List[Dict]) -> Set[str]:
+        names = [pg.get('name', f'pg{i+1}') for i, pg in enumerate(param_groups)]
```

Comment:
```suggestion
        names = [pg.get('name', f'pg{i}') for i, pg in enumerate(1, param_groups)]
```

Reply:
I can't believe I have made it this far w/out learning about this before: https://docs.python.org/3.8/library/functions.html#enumerate

Thanks!